composer.json

```diff
@@ -22,7 +22,7 @@
     ],
     "require": {
         "php": "^7.2|^8.0",
-        "honeybadger-io/honeybadger-php": "^2.17.0",
+        "honeybadger-io/honeybadger-php": "dev-check_ins_refactor",
```

Comment:
I assume you'll be changing this :)

Reply:
Yes! I had to release the php package and then update here.